app/src/main/res/layout/main_fragment.xml

```diff
@@ -132,21 +141,21 @@
         <LinearLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
+            android:background="#FFFFFFFF"
             android:orientation="vertical">
 
             <TextView
-                android:layout_width="wrap_content"
+                android:layout_width="402dp"
                 android:layout_height="wrap_content"
-                android:textColor="#000"
-                android:textSize="16dp"
-                android:layout_margin="10dp"
-                android:text="Country Cases" />
```

Comment:
Added color to the TextView to for better separation between ListView and TextView which is more aesthetically pleasing

Reply:
I disagree with using custom layout width instead of using wrap text because it will cause problems in different sizes of mobile devices

Reply:
Agree on that... better to keep the width flexible.